flatpak/README.md

```diff
@@ -24,6 +24,13 @@ flatpak remote-add --if-not-exists --user asimov-cli --no-gpg-verify https://asi
 flatpak install asimov-cli com.asimov_platform.asimov_cli
 ```
 
+## Updating
+
+To update to the latest version, simply run:
+```bash
+flatpak update com.asimov_platform.asimov_cli
```

Comment:
Note the requested app ID change: https://github.com/asimov-platform/asimov-packaging/pull/4#discussion_r2026697377 and update this PR after that's done in #4

Reply:
@artob it is done in - https://github.com/asimov-platform/asimov-packaging/pull/6/commits/2b4cc44db4bf44dbbf95c685e498ea04f2c642cf